astropy/io/votable/tests/test_table.py

```diff
@@ -261,6 +262,61 @@ def test_write_with_format():
     assert b"TABLEDATA" not in obuff
 
 
+@pytest.mark.skipif(not HAS_PYARROW, reason="requires pyarrow")
+def test_read_write_votable_parquet(tmp_path):
+    """
+    Test to write and read VOTable with Parquet serialization
+    """
+
+    # Create some fake data
+    number_of_objects = 10
+    ids = [f"COSMOS_{ii:03g}" for ii in range(number_of_objects)]
+    redshift = np.random.uniform(low=0, high=3, size=number_of_objects)
+    mass = np.random.uniform(low=1e8, high=1e10, size=number_of_objects)
+    sfr = np.random.uniform(low=1, high=100, size=number_of_objects)
+    astropytab = Table([ids, redshift, mass, sfr], names=["id", "z", "mass", "sfr"])
+
+    # Create Column metadata
+    column_metadata = {
+        "id": {"unit": "", "ucd": "meta.id", "utype": "none"},
+        "z": {"unit": "", "ucd": "src.redshift", "utype": "none"},
+        "mass": {"unit": "solMass", "ucd": "phys.mass", "utype": "none"},
+        "sfr": {"unit": "solMass / yr", "ucd": "phys.SFR", "utype": "none"},
+    }
+
+    # Write VOTable with Parquet serialization
+    filename = tmp_path / "test_votable_parquet.vot"
+    astropytab.write(
+        filename,
+        column_metadata=column_metadata,
+        overwrite=True,
+        format="votable.parquet",
+    )
+
+    # Open created VOTable with Parquet serialization
+    votable = parse(filename)
```

Comment:
It is failing here. Could it be that the parser cannot take Path object?

```suggestion
    votable = parse(str(filename))
```

Reply:
Hmm, I see another test here that passes Path into `parse` but it wasn't *this* serializer though. 🤷

Reply:
ahh, sorry, I didn't see this, but cooked up the same thing in the meantime. I also wonder why this never came up before (I followed the examples from the other Table.write tests, so was puzzled why we see the error)

Reply:
> I also wonder why this never came up before

Hmm, I am not sure either. Nothing pops into mind on any changes that would affect path parsing on Windows recently.

Reply:
I have to leave soon, but pushed one more debug commit to double check whether it's an existing bug. If it's, then we'll hack it around in this PR and I'll open a separate issue for it.

Reply:
The parser is a red herring, the bug is in the code in this branch, and relates to the absolute/relative path issue we have discussed already. (I can produce a very similar error locally on OSX).